### ISSUE TITLE
Screen reader support is lacking

### ISSUE THREAD

**Issue by jouni** [vaadin/vaadin-select#191]
The component (including the items in the dropdown) is announced as a generic “group”. 

The host component should be announced as a pop-up button.

The dropdown list should be announced as a menu and the items as menu items.

**Comment by zeitwesen** [comment]
- Also the label - given and inside a <vaadin-form-group> is not read by a screenreader.
- The dropdown-arrow has a hard-coded description "Toggle" - this should be customizable as well
